test/built-ins/TypedArray/prototype/forEach/callbackfn-detachbuffer.js

```diff
@@ -29,14 +29,11 @@ testWithTypedArrayConstructors(function(TA) {
   var loops = 0;
   var sample = new TA(2);
 
-  assert.throws(TypeError, function() {
-    sample.forEach(function() {
-      if (loops === 1) {
-        throw new Test262Error("callbackfn called twice");
-      }
+  sample.forEach(function() {
+    if (loops === 0) {
       $DETACHBUFFER(sample.buffer);
-      loops++;
-    });
+    }
+    loops++;
   });
 
   assert.sameValue(loops, 1);
```

Comment:
Looks like this one got missed.
```suggestion
  assert.sameValue(loops, 2);
```